### DISCUSSION TITLE
Parachuting" example in HemoCell

### DISCUSSION THREAD

**Discussion by Mansoor-Jadidi-2023** [UvaCsl/HemoCell#103] [General discussion]
Hello, Hemocell Community,

This is my first participation in this helpful discussion. I am interested in using the "Parachuting" example in HemoCell as a starting point. However, I would like to make some modifications. 
I have made some changes to the example. Firstly, I modified the orientation of the geometry in tube.stl (from x to y-direction). Consequently, I adjusted the values in the 'parachuting.cpp' file from **(poiseuilleForce, 0.0, 0.0)** to **(0.0, poiseuilleForce, 0.0)** and changed **hemocell.setSystemPeriodicity(0, true)** to **hemocell.setSystemPeriodicity(1, true)**.
Furthermore, I created a new RBC.pos in the packCells function as **"./packCells 10 20 10 --rbc 1"**. However, when visualising the simulation in Paraview, I am unable to see the RBC (attached file).

![Parachuting](https://github.com/UvaCsl/HemoCell/assets/138075870/476b9bc4-7fe1-4b61-a453-f5b1abb0eb80)

I would greatly appreciate it if someone could provide me with some advice or suggestions regarding this issue.

Thanks in advance,
Mansoor

**Comment by LGM-1** [comment]
Hello, Mansoor

May I ask how your problem was resolved? I seem to have encountered the same situation

Thanks in advance,
LGM-1
